wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -109,6 +112,17 @@ def check_attribute_name(self, node: ast.ClassDef) -> None:
                     naming.UpperCaseAttributeViolation(target, text=target.id),
                 )
 
+    @property
```

Comment:
It can be:
1. Cached
2. Moved to `logic/`

Reply:
> 1. Cached

Current implementation provide caching, it'a similar to `@cached_property` from `django`.
Do you want to introduce new dependency e.g. [`cached-property`](https://pypi.org/project/cached-property/) just for it?

> 2. Moved to `logic/`

I can move it there if you want, but could you tell me to which file or maybe I should create new one?

Reply:
You can create a new one 👍

Reply:
No new dependecies are required, you can try to use `lru_cache`

Reply:
However when the code responsible for creating variable names' blacklist will be moved to `logic/` then it should be enough to just assign the result in `__init__`:

```python
def __init__(...):
    ...
    self.variable_names_blacklist = logic.create_variable_names_blacklist()
```

What do you think?

Reply:
Sounds fine!